includes/models/model.llms.product.php

```diff
@@ -244,26 +351,20 @@ public function has_free_access_plan() {
 	 */
```

Comment:
missing changelog here, on purpose?

Reply:
obviously I didn't miss it on purpose. :-p